src/dal/db.ts

```diff
@@ -0,0 +1,14 @@
+export enum Database {
+    Settings = "settings",
+}
+
+// Loads the data from the database.
+export const Load = (database: Database): any => {
+    const data = localStorage.getItem(database);
+    if (!data) return undefined;
+    return JSON.parse(data);
+}
+
+export const Save = (database: Database, data: any) => {
+    localStorage.setItem(database, JSON.stringify(data));
+}
```

Comment:
can you make a global variable that implements method Load and Save, so later one we can replace it with other implementation easily (e.g. indexeddb, etc)